crates/krun-server/src/cli_options.rs

```diff
@@ -0,0 +1,38 @@
+use bpaf::{any, construct, env, positional, OptionParser, Parser};
+
+#[derive(Clone, Debug)]
+pub struct Options {
+    pub server_port: u32,
```

Comment:
```suggestion
    pub port: u32,
```

Reply:
I'm not entirely convinced about this suggestion. It's likely we'll need to eventually add support for redirecting ports with passt, and I feel the name `server_port` will avoid confusion in the future...

Reply:
Noted.